src/data/posts/twitter-and-the-perils-of-obedience.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Twitter and the Perils of Obedience
+excerpt: 'Reflections on the current state of Twitter, and its similarity to the shocking results of a 1961 experiment by Yale Researcher Stanley Milgram.'
+tags: [philosophy, twitter, vote, psychology]
+date: 11-28-22
+published: true
+path: twitter-and-the-perils-of-obedience
+---
+
+## Milgram's Experiment
+
+In 1961, a Yale University researcher named Stanley Milgram conducted a series of experiments as a part of his research on behavioral psychology. The goal of the experiment was to test participants' tendency to obey authority figures when under pressure.
+
+Throughout the course of the experiment, the experiment was conducted on 40 volunteers; all men between the ages of 20 and 50, with a variety of education and economic backgrounds.
```

Comment:
Confusing ... would reword to "The experiment was conducted on 40 volunteers; all men between the ages of 20 and 50, and each with a variety of education and economic backgrounds."